src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/PackagePropertyPage.xaml

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
+<Rule Name="PackagePropertyPage"
+      Description="Properties related to producing NuGet packages."
+      DisplayName="Package"
+      PageTemplate="generic"
+      xmlns="http://schemas.microsoft.com/build/2009/properties" >
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFileWithInterception"
+                SourceOfDefaultValue="AfterContext"
+                HasConfigurationCondition="False" />
+  </Rule.DataSource>
+
+  <BoolProperty Name="GeneratePackageOnBuild"
+                DisplayName="Generate NuGet package on build" />
+
+  <BoolProperty Name="PackageRequireLicenseAcceptance"
+                DisplayName="Require license acceptance" />
+
+  <StringProperty Name="PackageId"
+                  DisplayName="Package ID"
+                  Description="The case-insensitive package identifier, which must be unique across nuget.org or whatever gallery the package resides in. IDs may not contain spaces or characters that are not valid for a URL, and generally follow .NET namespace rules."
+                  HelpUrl="https://docs.microsoft.com/nuget/create-packages/creating-a-package#choose-a-unique-package-identifier-and-setting-the-version-number" />
```

Comment:
How do we handle these links breaking? Something like aka.ms would allow us to fix breakages without updating VS.

Reply:
Good point. I'll look into it.